test/DriverTest.php

```diff
@@ -85,7 +115,7 @@ public function testLstat(): \Generator
 
         $target = "{$fixtureDir}/small.txt";
         $link = "{$fixtureDir}/symlink.txt";
-        $this->assertTrue(yield File\symlink($target, $link));
+        yield File\symlink($target, $link);
```

Comment:
This could be changed to `$this->assertNull`

Reply:
I'm using assertNull in testSymlink. This is not a test for the symlink method so I omitted it on purpose. Do you want it here anyway?

Reply:
No, fine then.